init/src/init.ts

```diff
@@ -411,7 +411,7 @@ export default define.page(function Home() {
   const count = useSignal(3);
 
   return (
-    <div class="px-4 py-8 mx-auto fresh-gradient">
+    <div class="px-4 py-8 mx-auto fresh-gradient min-h-screen">
```

Comment:
Nice, can you also add the CSS for that to the fallback code for when people create a project without tailwindcss? The code for that is in the variable `NO_TAILWIND_STYLES`

Reply:
Ah not quite. The fallback css code is _only_ loaded when you don't pick the "use tailwindcss" option. When you _do_ pick tailwind this will not be used, so it should be a tailwind class name regardless. Tailwind doesn't have a `full-height` class as far as I'm aware.